metacat-common/src/main/java/com/netflix/metacat/common/json/MetacatJsonLocator.java

```diff
@@ -113,7 +113,7 @@ public ObjectNode parseJsonObject(final String s) {
         } else if (node.isObject()) {
             return (ObjectNode) node;
         } else {
-            throw new MetacatJsonException("Cannot convert '" + s + "' to a json object");
+            throw new MetacatJsonParseException(s, "Cannot convert '" + s + "' to a json object");
```

Comment:
What is the reason to add a class, instead of creating another constructor in MetacatJsonException